docs/man_pages/project/testing/livesync-android.md

```diff
@@ -1,21 +1,36 @@
 livesync android
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder livesync android [--device <Device ID>] [--companion] [--watch]`
 
 Synchronizes the latest changes in your project to connected Android devices.
 
+<% if(isConsole) { %>
+<% if(isMobileWebsite) { %>
+WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help livesync android`
+<% } %>
+<% if(isNativeScript) { %>
+WARNING: This command is not applicable to NativeScript projects. To view the complete help for this command, run `$ appbuilder help livesync android`
```

Comment:
I will fix this but when this huge a bug exists, it is a limitation that needs to be noted as a limitation in the command.